### PR TITLE
`@var` can be replaced with assert() call.

### PR DESCRIPTION
**Pull request by voku**
-> see this discussion -> https://github.com/doctrine/coding-standard/pull/47

---
This change is [Reviewable](https://reviewable.io/reviews/kubawerlos/php-cs-fixer-custom-fixers/91)

### CODE REVIEW

File: .php_cs.dist [hunk 1]
```diff
@@ -2,6 +2,8 @@
 
 declare(strict_types = 1);
 
+require_once __DIR__ . '/vendor/autoload.php';
```
Comment (kubawerlos) [inline .php_cs.dist]: What is the benefit of adding this?

File: phpstan.neon.dist [hunk 2]
```diff
@@ -6,5 +6,7 @@ parameters:
         - '#^Method PhpCsFixerCustomFixers\\Fixer\\OperatorLinebreakFixer::getOperatorIndices\(\) should return array<int>\|null but returns array<int, int\|null>\.$#'
         - '#^Only numeric types are allowed in (\-|\+|post\-decrement|post\-increment), int\|null given( on the left side)?\.$#'
         - '#^Parameter .+ expects .+, .+ given\.$#'
+        - '#Call to function assert\(\) with true will always evaluate to true\.#'
```
Comment (kubawerlos) [inline phpstan.neon.dist]: No need for this anymore I guess.

File: .php_cs.dist (hunk 1 above)
Comment (voku) [reply .php_cs.dist]: mmh... I think I was running php-cs fixer via phar file ... sorry I will remove it.

File: phpstan.neon.dist (hunk 2 above)
Comment (voku) [reply phpstan.neon.dist]: I will test it ...

**Comment by voku** [comment]
@kubawerlos Hi, have you any additional idea how to improve this fixer? Maybe some special test cases?

PS: thanks for this awesome collection of fixer

**Comment by kubawerlos** [comment]
Hi @voku, can you update (most likely rebase) PR to allow reviewing it?

**Comment by kubawerlos** [comment]
Replaced by https://github.com/kubawerlos/php-cs-fixer-custom-fixers/pull/675.

**Pull request closed without merging**